etl/types/trasnformer_impl.go

```diff
@@ -0,0 +1,186 @@
+// Copyright 2020 Insolar Network Ltd.
+// All rights reserved.
+// This material is licensed under the Insolar License version 1.0,
+// available at https://github.com/insolar/block-explorer/blob/master/LICENSE.md.
+
+package types
+
+import (
+	"github.com/insolar/block-explorer/utils"
+	"github.com/insolar/insolar/insolar"
+	ins_record "github.com/insolar/insolar/insolar/record"
+	"github.com/insolar/insolar/ledger/heavy/exporter"
+	"github.com/pkg/errors"
+)
+
+type Transform interface {
+	Transform() (*JetDrop, error)
+}
+
+const (
+	// delta between pulses
+	pulseDelta uint16 = 10
+)
+
+// Transform transforms thr row JetDrops to canonical JetDrops
+func (jd *PlatformJetDrops) Transform() ([]*JetDrop, error) {
```

Comment:
Why method on struct and not function with jetDrop as param? This way we could store all transform logic in transform package